### ISSUE TITLE
False positive on "Large numbers" when it doesn't follow expected pattern

### ISSUE THREAD

**Issue by brainlid** [rrrene/credo#507]
### Environment

* Credo version (`mix credo -v`): 0.8.10
* Erlang/Elixir version (`elixir -v`): 1.5.2
* Operating system: Ubuntu 17.10

### What were you trying to do?

Running the credo tests.

Value like this...

```elixir
  @max_limit 99_999_00
```
Where the value represents cents in US currency. Normally I would write it as "99,999.00".

I am using using the underscores to increase readability, but credo wants it to be "9_999_900". This is correct, that's how many "cents" it is. It's more readable for me to put the underscores in different places. Elixir is fine with the alternate placement.

I disabled the check for this line.

### Expected outcome

No suggestion or warning.

### Actual outcome

```
Large numbers should be written with underscores: 9_999_900
```

**Comment by rrrene** [comment]
@brainlid Thx for this report. I see your problem.

How would we support this use case? Maybe we add a config parameter for this. I am open to ideas on how to do this! :+1:

**Comment by brainlid** [comment]
Thanks René,

I guess it depends on how the check is currently implemented. If it's a
Regex, perhaps it's too strict? Like just check that there are "_"
underscore characters in a sequence of large numbers? Just don't expect or
enforce the position and grouping of the numbers?

Not sure if that makes sense for how the check is performed.

Thanks!

**Comment by yukster** [comment]
I also denote my dollar amounts ending in '_00'. It would be great if the check just made sure a large number had underscores but without caring about the position.

**Comment by rrrene** [comment]
I would be open to merge a pull request implementing this proposal. :+1:
